big_tests/tests/mod_event_pusher_rabbit_SUITE.erl

```diff
@@ -582,6 +590,13 @@ stop_mod_event_pusher_rabbit() ->
     Host = ct:get_config({hosts, mim, domain}),
     rpc(mim(), gen_mod, stop_module, [Host, mod_event_pusher_rabbit]).
 
+start_wpool(Host) ->
```

Comment:
The function is misleading. `mongoose_wpool` is already running. What is started here is only the rabbit pool.